libs/application/templates/accident-notification/src/forms/AccidentNotificationForm.ts

```diff
@@ -428,6 +428,21 @@ export const AccidentNotificationForm: Form = buildForm({
                     },
                   ],
                 }),
+                buildCustomField(
+                  {
+                    id: 'attachments.injuryCertificate.alert',
+                    title: attachments.labels.alertMessage,
+                    description:
+                      accidentType.warning.agricultureAccidentWarning,
+                    component: 'FieldAlertMessage',
+                    condition: (formValue) =>
+                      (formValue as {
+                        workAccident: { type: WorkAccidentTypeEnum }
+                      }).workAccident?.type ===
+                      WorkAccidentTypeEnum.AGRICULTURE,
+                  },
+                  { type: 'warning' },
```

Comment:
Don't we have a utils function for this?

Reply:
We do, good spot!